pandas/tests/test_strings.py

```diff
@@ -2966,23 +2966,21 @@ def test_partition_with_name(self):
         assert res.nlevels == 1
         tm.assert_index_equal(res, exp)
 
-    def test_partition_deprecation(self):
+    def test_partition_sep_kwarg(self):
         # GH 22676; depr kwarg "pat" in favor of "sep"
         values = Series(["a_b_c", "c_d_e", np.nan, "f_g_h"])
 
         # str.partition
         # using sep -> no warning
         expected = values.str.partition(sep="_")
-        with tm.assert_produces_warning(FutureWarning):
-            result = values.str.partition(pat="_")
-            tm.assert_frame_equal(result, expected)
+        result = values.str.partition("_")
+        tm.assert_frame_equal(result, expected)
 
         # str.rpartition
         # using sep -> no warning
```

Comment:
Small nit, but those comments are no longer up to date (also above)

Reply:
updated #29985 to remove these comments